canary/webrtc-c/jobs/runner.groovy

```diff
@@ -254,24 +353,27 @@ pipeline {
                 // TODO: Maybe there's a better way to write this instead of duplicating it
                 build(
                     job: env.JOB_NAME,
+
                     parameters: [
+                      string(name: 'AWS_DEFAULT_REGION', value: params.AWS_DEFAULT_REGION),
                       string(name: 'AWS_KVS_LOG_LEVEL', value: params.AWS_KVS_LOG_LEVEL),
                       booleanParam(name: 'IS_SIGNALING', value: params.IS_SIGNALING),
+                      booleanParam(name: 'IS_STORAGE', value: params.IS_STORAGE),
                       booleanParam(name: 'USE_TURN', value: params.USE_TURN),
                       booleanParam(name: 'USE_IOT', value: params.USE_IOT),
-                      booleanParam(name: 'IS_PROFILING', value: params.IS_PROFILING),
```

Comment:
Same here. 
I think when you rebased, you only took the PR branch changes and removed master branch changes. It should have been the other way round to include the master branch changes in your branch.